src/io.rs

```diff
@@ -163,6 +236,10 @@ fn catch<'vm>(
     })
 }
 
+fn throw(msg: String) -> IO<OpaqueValue<RootedThread, A>> {
```

Comment:
This actually already exists as the `error` function. Do you feel there is a good reason to have it here as well?

Reply:
`error` is `String -> a` though, not `String -> IO a`, or am I missing something? Name's probably not the best, just added it quickly to make things work.